addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/spider.html

```diff
@@ -51,40 +51,54 @@ <h4>HTML</h4>
 
 	<h4>Robots.txt file</h4>
 	If set in the
-	<a href="options.html">Options Spider
-		screen</a>, it also analyzes the 'Robots.txt' file and tries to identify
+	<a href="options.html">Options Spider screen</a>, it also analyzes 
+	the 'Robots.txt' file and tries to identify
 	new resources using the specified rules. It has to be mentioned that
 	the Spider does not follow the rules specified in the 'Robots.txt'
 	file.
 
-	<h4>SVG Files</H4>
-	SVG image files are parsed to identify HREF attributes and extract/resolve any contained links.
+	<h4>sitemap.xml file</h4>
+	If set in the <a href="options.html">Options Spider screen</a>, 
+	the Spider also analyzes the 'sitemap.xml' file and try to identify
```

Comment:
`tries`

Reply:
I really think `try` reads more correctly here.

Reply:
The "Spider try".